kokoro/release/python/macos/config.sh

```diff
@@ -0,0 +1,50 @@
+# Define custom utilities
+# Test for OSX with [ -n "$IS_OSX" ]
+
+function pre_build {
+    # Any stuff that you need to do before you start building the wheels
+    # Runs in the root directory of this repository.
+    pushd protobuf
+
+    export CC=gcc
+    export CXX=g++
+    # Build protoc
```

Comment:
No need to brew install now?

Reply:
Still trying to verify that this actually works. It is still running...